dockers/docker-dummyk8s/Dockerfile.j2

```diff
@@ -0,0 +1,19 @@
+FROM docker-config-engine-bullseye-{{DOCKER_USERNAME}}:{{DOCKER_USERTAG}}
+
+ARG docker_container_name
+ARG image_version
+
+# Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Pass the image_version to container
+ENV IMAGE_VERSION=$image_version
+
+# Update apt's cache of available packages
+RUN apt-get update
+
+COPY ["readiness-probe.sh", "/usr/bin"]
+
+RUN chmod +x /usr/bin/readiness-probe.sh
+
+ENTRYPOINT ["tail", "-f", "/dev/null"]
```

Comment:
should we just sleep instead? maybe just sleep for 15 minutes so that we can also verify that k8s can restart our container?

https://stackoverflow.com/questions/43843079/using-tail-f-dev-null-to-keep-container-up-fails-unexpectedly

Reply:
Thanks, updated to sleep instead